package.json

```diff
@@ -110,9 +110,10 @@
     "jssha": "^2.3.1",
     "jwt-decode": "^2.2.0",
     "libxmljs2": "^0.21.3",
+    "lodash": "^4.17.10",
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `lodash` violates your company's policies.

Lifecycle recommends using version [4.17.12](https://www.npmjs.com/package/lodash/v/4.17.12) instead as it does not violate any policies.